torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -266,23 +266,23 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
     Function& m,
     const std::string& field) {
   // 1. Look inside script::Module object for the field.
-  const auto& selfType = concreteType_->getJitType();
+  const auto& selfType_ = concreteType_->getJitType();
+  if (selfType_->cast<InterfaceType>()) {
+    return std::make_shared<SimpleValue>(self_)->attr(loc, m, field);
```

Comment:
`SimpleValue(self)->` since we are deleting the object directly after.

Reply:
Changing it would cause a lint error internally: https://fburl.com/wiki/jshlq9g3. Which doesn't apply in this situation but I guess could conceivably happen with some more code mutations down the road, so we might as well follow the rule?